.gitignore.rb

```diff
@@ -10,5 +10,8 @@
 
     # Ignore the test coverage results from SimpleCov
     /coverage
+
+    # Ignore generated OpenAPI file
+    /public/openapi.yml
```

Comment:
This should only be added if the user selected the `Open API` addon. Let's create a `.gitignore.rb` file inside the `addons/openapi` folder 👍 

(example of implementation is available as a similar implementation is done for the web variant: `.template/variants/web/.gitignore.rb`)